shared/reducers/tracker.js

```diff
@@ -446,11 +463,11 @@ function remoteProofToProof (rp: RemoteProof, lcr: ?LinkCheckResult): Proof {
     state: proofState,
     id: rp.sigID,
     meta: statusMeta || diffMeta,
-    type: proofType,
+    type: remoteProofToProofType(rp),
     color: stateToColor(proofState),
     name: rp.displayMarkup,
     humanUrl: humanUrl,
-    profileUrl: rp.displayMarkup && proofUrlToProfileUrl(rp.proofType, rp.displayMarkup, rp.key, humanUrl)
+    profileUrl: rp.displayMarkup && proofUrlToProfileUrl(rp.proofType.toString(), rp.displayMarkup, rp.key, humanUrl)
```

Comment:
@MarcoPolo Here's the flow error if I don't:

```
 λ flow

reducers/tracker.js:470
470:     profileUrl: rp.displayMarkup && proofUrlToProfileUrl(rp.proofType, rp.displayMarkup, rp.key, humanUrl)
                                                              ^^^^^^^^^^^^ number literal `0`. This type is incompatible with
412: function proofUrlToProfileUrl (proofType: string, name: string, key: ?string, humanUrl: ?string): string {
                                               ^^^^^^ string

[.. more errors for the other numbers in the enum]

reducers/tracker.js:470
470:     profileUrl: rp.displayMarkup && proofUrlToProfileUrl(rp.proofType, rp.displayMarkup, rp.key, humanUrl)
                                                              ^^^^^^^^^^^^ number literal `6`. This type is incompatible with
412: function proofUrlToProfileUrl (proofType: string, name: string, key: ?string, humanUrl: ?string): string {
                                               ^^^^^^ string


Found 11 errors
```

But I don't see why my change to create remoteProofToProofType() provoked the error.  Any ideas, something else I should try?